File: android/src/main/kotlin/ru/kardenvan/flutter_yandex_ads_plus/ads/native_yandex_ad/NativeYandexAdViewArguments.kt
package ru.kardenvan.flutter_yandex_ads_plus.ads.native_yandex_ad

import android.util.Size
import ru.kardenvan.flutter_yandex_ads_plus.ads.AdParameters

data class NativeYandexAdViewArguments(
    val viewUid: String,
    val adId: String,
    val minSize: Size,
    val theme: NativeYandexAdViewTheme,
    val parameters: AdParameters?
)

File: ios/Classes/ads/native_yandex_ad/NativeYandexAdViewArguments.swift
//
//  BannerViewArguments.swift
//  flutter_yandex_ads_plus
//
//  Created by apple on 21.10.2022.
//

import Foundation

class NativeYandexAdViewArguments {
    let uid: String
    let adUid: String
    let height: Int
    let width: Int
    let theme: NativeYandexAdViewTheme
    let parameters: AdParameters?
    
    init(
        uid: String,
        adId: String,
        height: Int,
        width: Int,
        theme: NativeYandexAdViewTheme,
        parameters: AdParameters?
    ) {
        self.uid = uid
        self.adUid = adId
        self.height = height
        self.width = width
        self.theme = theme
        self.parameters = parameters
    }
}
